c/include/ml-api-service.h

```diff
@@ -190,6 +190,37 @@ int ml_service_destroy (ml_service_h handle);
  */
 int ml_service_get_pipeline_state (ml_service_h handle, ml_pipeline_state_e *state);
 
+/**
+ * @brief Creates query service handle with given ml-option handle.
+ * @since_tizen 7.0
+ * @remarks The @a handle should be destroyed using ml_service_destroy().
+ * @param[in] option The option used for creating query service.
+ * @param[out] handle Newly created query service handle is returned.
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ */
+int ml_service_query_create (ml_option_h option, ml_service_h *handle);
+
+/**
+ * @brief Requests query service an output with given input data.
```

Comment:
It's technically correct, but might sound better like this "Requests the query service to process the @a input tensor and produce an @a output tensor." OR "Requests the query service to compute @output data from the given @a input." (the word "compute" can be replaced by something more appropriate for ML)

Reply:
Thanks! I changed the sentence with your concise suggestion, PTAL:
"@brief Requests the query service to process the @a input and produce an @a output."